bot/resources/tags/identity.md

```diff
@@ -0,0 +1,24 @@
+**Identity vs. Equality**
+
+Should I be using `is` or `==`?
+
+To check if two things are equal, use the equality operator (`==`).
+```py
+x = 5
+if x == 5:
+    print("x equals 5")
+if x == 3:
+    print("x equals 3")
+# Prints 'x equals 5'
+```
+To check if two things are actually the same thing in memory, use the identity comparison operator (`is`).
+```py
+x = [1, 2, 3]
+y = [1, 2, 3]
+if x is [1, 2, 3]:
+    print("x is y")
+z = x
```

Comment:
This snippet could get more explicit variable names too, it is a bit hard to follow right now. Also using the integer `420` would be less noisy than `[1, 2, 3]` and keep the same behavior.

Reply:
I think there's merit in using specifically lists, as it is something that confuses a lot of people.

For integers and strings this works only if the value is cached and shouldn't be relied on.

Reply:
I've kept it using the list given what zig said